src/app/public/modules/modal/modal.component.scss

```diff
@@ -148,3 +153,39 @@
   min-height: 460px;
   margin: -$sky-margin-plus-half;
 }
+
+@include sky-theme-modern {
+  .sky-modal-header {
+    border: none;
+    padding: $sky-theme-modern-space-lg $sky-theme-modern-space-xl;
+  }
+
+  .sky-modal-btn-help,
+  .sky-modal-btn-close {
+    display: none;
```

Comment:
Now that we're hiding the close button for modern theme, does this present any accessibility issues? How does a someone using keyboard controls know how to close the modal?

Reply:
Modern theme use cases would always provide a button in the modal footer for closing the modal. The `Escape` key still works for closing the modal using the keyboard.